packages/ilib-loctool-webos-dart/test/integrationTest/DartApp.test.js

```diff
@@ -111,14 +111,15 @@ describe('[integration] test the localization result of webos-dart app', () => {
         expect(jsonData["Search"]).toBe("통합 검색");
     });
     test("dartsample_test_es_CO", function() {
-        expect.assertions(4);
+        expect.assertions(5);
```

Comment:
How about adding examples to `samples-loctool/webos-dart` as well? The integration tests included in the package provide a very simplified version, while validation for various cases is available in sample-loctool.